### DISCUSSION TITLE
Basic log in web page

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#3093] [General]
I think, it could be useful to have the command-line log as well under e.g. Configuration.
When something goes wrong; or at least looks like going wrong, I'll ssh into the box and `$ cat logs/2.1.0-28-g9dd54a98 ` or likewise. I think it would be good to be able to access the basic log as well from the web interface.

**Comment by whyman** [comment]
Logs in the UI is a great idea but I wonder if it will mean we don't work on nicer ways to show errors like an "import issues" screen or something like that. I think we do a bad job of recording and surfacing that sort of thing currently

But yeah it would make it easier to check from your phone

**Reply by udippel** [reply]
Just trying to come up with something useful ... .
I have no idea about man-months available, neither on long-term milestones. 
If I still had students, I'd give out the task of writing an app. That's where this world moves to: apps. An app could as well be used as wrapper for BubbleUPnP: administration and usage.

For me still the primary target number one, however, is the merging of two different configurations and the lack of exporting and writing back of configurations. Like what any cheap linux-based box from WiFi-client upwards does. Unifying the configuration and making it exportable so that we can write the config back after a reinstall, loss or deletion of database, and especially for any chance of a roll-out of this software; aside of repackaging into a .deb or so. 
gerbera is so comfortable, so configurable, so versatile - though principally always from scratch (aside of copying the config.xml which is limited in its handling). It is ro in some locations, which has advantages and disadvantages, had no mirror in case of rw (like the database). In a nutshell: configuration to me still looks like the step-child of the project. 
The new first page pays tribute to this, and works around this '2-config-quagmire'. It ought to be different: You start playing with it, all in-flight. Whenever you get what you want, you can save what you have ("I_love_this_gerbera.xml"). You keep on playing and you like it: you save it as "I_Love_this_2.xml". You don't like it: you revert to the previous one. 
(Default names being the time stamps, though, of course.)
